### PR TITLE
Allow 640

### PR DESCRIPTION
**Pull request by inaod568**
# Description

Please include the following in this section

- [ ] Summary of the changes and the related issue
- [x] Relevant motivation and context
- [ ] Any dependencies that are required for this change

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

Please delete options that are not relevant.

- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Rebase/merge the branch which this PR is made to
- [ ] Product Owner / Scrum Master: This PR is made to the `master` branch and I have updated the [version](../dds_cli/version.py)

## Formatting and documentation

- [x] I have added a row in the [changelog](../CHANGELOG.md)
- [x] The code follows the style guidelines of this project: Black / Prettier formatting
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Tests

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: dds_cli/user.py [hunk 1]
```diff
@@ -298,9 +300,13 @@ def check_token_file_permissions(self):
         """
         if os.name != "nt":
             st_mode = os.stat(self.token_file).st_mode
+            print(f"st_mode: {st_mode}")
+            print(f"stat.S_IMODE(st_mode): {stat.S_IMODE(st_mode)}")
```
Comment (inaod568) [inline dds_cli/user.py:304]:
```suggestion
```

File: dds_cli/user.py [hunk 2]
```diff
@@ -298,9 +300,13 @@ def check_token_file_permissions(self):
...
             permissions_octal = oct(stat.S_IMODE(st_mode))
+            print(f"permissions_octal: {permissions_octal}")
```
Comment (inaod568) [inline dds_cli/user.py:306]:
```suggestion
```

File: dds_cli/user.py [hunk 3]
```diff
@@ -298,9 +300,13 @@ def check_token_file_permissions(self):
...
             permissions_readable = stat.filemode(st_mode)
-            if permissions_octal != "0o600":
+            print(f"permissions_readable: {permissions_readable}")
```
Comment (inaod568) [inline dds_cli/user.py:308]:
```suggestion
```

**Review by valyo — APPROVED** [review]
Looks fine and works as expected

**Pull request merged**